packages/asset/contracts/mock/MockMinter.sol

```diff
@@ -1,8 +1,8 @@
 //SPDX-License-Identifier: MIT
 pragma solidity 0.8.18;
 
-// import IAsset from "./IAsset.sol";
-import "../libraries/TokenIdUtils.sol";
+import {IAsset} from "../interfaces/IAsset.sol";
```

Comment:
I don't think this is being used in this contract, lets either use it or remove it.

Reply:
Both the library and Interface are being used in the contract

Reply:
My bad, disregard.